### PR TITLE
Style cleanup of `modules/assim.batch/*.R` scripts

### PR DESCRIPTION
**Pull request by bpbond**
There was a _lot_ of code here.

 Ran through formatR::tidy_dir with arrow=TRUE and indent=2; made sure all if blocks have braces. Tried to make sure formatR didn’t make things too jaggy. Substituted `seq_len`, `seq_along`, `paste0`, and `file.path`.

Note lines 357-359 in original `pda.utils.R`, which I have corrected to
```R
ifelse(log, p, exp(p))
```

### CODE REVIEW

File: modules/assim.batch/R/pda.utils.R [hunk 1]
```diff
@@ -334,41 +341,42 @@ pda.create.ensemble <- function(settings, con, workflow.id) {
...
-      pprior[[i]] <- parse(text=paste("pexp(q,",prior$parama[i],")",sep=""))
-    }else{
-      dprior[[i]] <- parse(text=paste("d",prior$distn[i],"(x,",prior$parama[i],",",prior$paramb[i],",log=TRUE)",sep=""))
-      rprior[[i]] <- parse(text=paste("r",prior$distn[i],"(n,",prior$parama[i],",",prior$paramb[i],")",sep=""))
-      qprior[[i]] <- parse(text=paste("q",prior$distn[i],"(p,",prior$parama[i],",",prior$paramb[i],")",sep=""))
-      pprior[[i]] <- parse(text=paste("p",prior$distn[i],"(q,",prior$parama[i],",",prior$paramb[i],")",sep=""))
+  dprior <- rprior <- qprior <- pprior <- list()
+  for (i in seq_len(n.param.all)) {
+    if (prior$distn[i] == "exp") {
+      dprior[[i]] <- parse(text = past0e("dexp(x,", prior$parama[i], ",log=TRUE)"))
```
Comment (istfer) [inline modules/assim.batch/R/pda.utils.R]: don't know what happened here with `past0e`

File: modules/assim.batch/R/pda.emulator.R [hunk 2]
```diff
@@ -9,397 +9,417 @@
...
       
       # Convert parameter values to probabilities according to previous prior distribution
       knots.list$probs <- knots.list$params
-      for(pft in seq_along(settings$pfts)){
-        for(i in 1:n.param.all[[pft]]) {
-          knots.list[[pft]]$probs[,i] <- eval(prior.fn[[pft]]$pprior[[i]], list(q=knots.list[[pft]]$params[,i]))
+      for (pft in seq_along(settings$pfts)) {
+        for (i in 1:n.param.all[[pft]]) {
+          knots.list[[pft]]$probs[, i] <- eval(prior.fn[[pft]]$pprior[[i]], list(q = knots.list[[pft]]$params[, 
+                                                                                                              i]))
```
Comment (istfer) [inline modules/assim.batch/R/pda.emulator.R]: there were a few funny cuts like this one, but I can tidy them up later

**Comment by istfer** [comment]
conflicts need to be resolved, otherwise ready

File: modules/assim.batch/R/pda.load.data.R [hunk 3]
```diff
@@ -13,43 +13,50 @@
 load.pda.data <- function(settings, con) {
   
   library(PEcAn.benchmark)
+<<<<<<< HEAD
+
```
Comment (istfer) [inline modules/assim.batch/R/pda.load.data.R]: conflict

File: modules/assim.batch/R/pda.mcmc.recover.R [hunk 4]
```diff
@@ -13,6 +13,15 @@
 # read.settings(path/to/original/settings/file)
 # settings$assim.batch <- pda.mcmc.recover(settings) # wrap up unfinished run
 # settings$assim.batch <- pda.mcmc(settings) # start new pda
+<<<<<<< HEAD
+pda.mcmc.recover <- function(settings, params.id = NULL, param.names = NULL, prior.id = NULL,
```
Comment (istfer) [inline modules/assim.batch/R/pda.mcmc.recover.R]: conflict

File: modules/assim.batch/R/pda.utils.R [hunk 5]
```diff
@@ -9,39 +9,48 @@
 ##' @export
 assim.batch <- function(settings) {
   # Quit if pda not requested in settings
-  if(!('assim.batch' %in% names(settings))) {
+  if (!("assim.batch" %in% names(settings))) {
     return(settings)
   }
+<<<<<<< HEAD
+=======
```
Comment (istfer) [inline modules/assim.batch/R/pda.utils.R]: conflict

File: modules/assim.batch/R/pda.utils.R [hunk 6]
```diff
@@ -578,39 +596,74 @@ pda.get.model.output <- function(settings, run.id, inputs) {
   
   library(PEcAn.benchmark)
   library(lubridate)
+<<<<<<< HEAD
+
```
Comment (istfer) [inline modules/assim.batch/R/pda.utils.R]: conflict

File: modules/assim.batch/R/pda.utils.R [hunk 7]
```diff
@@ -578,39 +596,74 @@ pda.get.model.output <- function(settings, run.id, inputs) {
...
+                                       end.year, 
+                                       variables = vars))
+    
+    if (length(model) == 0) {
+      # Probably indicates model failed entirely
       return(NA)
     }
     
+<<<<<<< HEAD
+    # FC - NEE specific hack 2: change NEE back to FC only if 'FC' was specified in the first place
```
Comment (istfer) [inline modules/assim.batch/R/pda.utils.R]: conflict

File: modules/assim.batch/R/plot.da.R [hunk 8]
```diff
@@ -2,6 +2,41 @@
 ## ported by M. Dietze 08/30/12
 ## some of this is redundant with other parts of PEcAn and needs to be cleaned up 
 
+<<<<<<< HEAD
+plot.da <- function(prior.dir, prior.file, in.dir, out.dir, next.run.dir) {
```
Comment (istfer) [inline modules/assim.batch/R/plot.da.R]: conflict

File: modules/assim.batch/R/plot.da.R [hunk 9]
```diff
@@ -111,117 +146,163 @@ for(i in 1:ncol(samp[[1]])) {
...
+      cv.post <- cv(samp.i[[rownames(post.distns)[[i]]]])
+      foo[da.post.i, ] <- c(pft, rownames(post.distns)[[i]], 
+                            cv.prior1, 
+                            cv.prior2,
+                            cv.post, 
+                            (cv.prior1 - cv.prior2) / cv.prior1, 
+                            (cv.prior2 - cv.post)/cv.prior2, 
+                            (cv.prior1 - cv.post)/cv.prior1)
+      post.distns[i, ] <- approx.posterior(samp.i, post.distns[i, ])
+      da.post.i <<- da.post.i + 1
```
Comment (istfer) [inline modules/assim.batch/R/plot.da.R]: double `<`

Comment (bpbond) [inline modules/assim.batch/R/plot.da.R]: Whoah, sorry about that.

**Comment by mdietze** [comment]
@istfer this one's still flagged as review required

**Comment by istfer** [comment]
@mdietze I already approved. I don't have admin permissions for the repo, can I still change the big red "Review required" flag ?

**Pull request merged**